packages/core/admin/admin/src/content-manager/pages/App/LeftMenu/index.js

```diff
@@ -8,23 +8,27 @@ import React, { useMemo, useState } from 'react';
 import { useSelector, shallowEqual } from 'react-redux';
 import { useIntl } from 'react-intl';
 import matchSorter from 'match-sorter';
+import { NavLink } from 'react-router-dom';
+
 import sortBy from 'lodash/sortBy';
 import toLower from 'lodash/toLower';
-import { NavLink } from 'react-router-dom';
+import camelCase from 'lodash/camelCase';
+
 import {
   SubNav,
   SubNavHeader,
   SubNavSection,
   SubNavSections,
   SubNavLink,
 } from '@strapi/design-system/v2/SubNav';
+
 import getTrad from '../../../utils/getTrad';
 import { makeSelectModelLinks } from '../selectors';
 
 const matchByTitle = (links, search) =>
   search
     ? matchSorter(links, toLower(search), { keys: [(item) => toLower(item.title)] })
```

Comment:
can we swap out `toLower` for the native impl `toLowerCase`? 🤔 
Feels unnecessary using lodash so much.